src/Hocon.Configuration/Config.cs

```diff
@@ -18,82 +20,63 @@ namespace Hocon
     ///     configuration string.
     /// </summary>
     [Serializable]
-    public class Config : HoconRoot, ISerializable, IEquatable<Config>
+    public class Config : HoconObject, ISerializable, IEquatable<Config>
     {
-        private static readonly HoconValue EmptyValue;
-
-        private const string SerializedPropertyName = "_data";
+        /// <summary>
+        ///     Identical to <see cref="HoconConfigurationFactory.Empty" />.
+        /// </summary>
+        /// <remarks>
+        ///     Added for brevity and API backwards-compatibility with Akka.Hocon.
+        /// </remarks>
+        public new static Config Empty { get; } = new Config();
```

Comment:
`new` keyword is needed here because this hides `HoconObject.Empty`?

Reply:
yes, because a new `Config` is different than a new `HoconObject`.